Node List/Node List/ReusableViews/NodeList/NodeListView.swift

```diff
@@ -0,0 +1,69 @@
+//
+//  NodeList.swift
+//  Node List
+//
+//  Created by Bryan Luna on 2/10/24.
+//
+
+import SwiftUI
+
+struct NodeListView: View {
+
+    @Binding private var selectedItem: Item?
+    @State private var selectedSubItem: Item?
+    
+    private let items: [Item]
+    
+    init(items: [Item], 
+         selectedItem: Binding<Item?>) {
+        self.items = items
+        _selectedItem = selectedItem
+        self.selectedSubItem = selectedItem.wrappedValue
+    }
+    
+    var body: some View {
+        ScrollView {
+            ForEach(items) { item in
+                LazyVStack {
+                    Button {
+                        selectedItem = item
```

Comment:
Whe you comparate the all item you need add the equaltable protocol to the model but is better only comparate that with ID

Reply:
🤔 
What do you mean?